lib/travis/api/v3/services/cron/create.rb

```diff
@@ -0,0 +1,24 @@
+module Travis::API::V3
+  class Services::Cron::Create < Service
+    result_type :cron
+    params :interval, :disable_by_build
+
+    def run!
+      raise LoginRequired unless access_control.logged_in? or access_control.full_access?
+      raise NotFound      unless repository = find(:repository)
+      raise NotFound      unless branch = find(:branch, repository)
+      raise ArgumentError, 'interval must be "daily", "weekly" or "monthly"' unless ["daily", "weekly", "monthly"].include?(params["interval"])
```

Comment:
I have not seen any usage of `WrongParameter`, I change it now to `V3:Error` (see 6fb0f7b6cef900e88c911a423805aad447b458dd)